src/app/SBE/lane/index.js

```diff
@@ -16,12 +16,11 @@ const Lane = ({ sbeCode, dashboard, width }) => {
     <Card
       variant="card0"
       p={[2, 2, 2, 3]}
-      width={width}
```

Comment:
we need width for the detail view, some of them use it. Shouldn't have any affect on the process view